anna/examples/simple-put-get/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "simple-put-get"
+version = "0.1.0"
+edition = "2021"
+license = "Apache-2.0"
+
+[dependencies]
+eyre = "0.6.5"
+chrono = { version = "0.4.19", features = ["serde"] }
+tokio = { git = "https://github.com/WasmEdge/tokio.git", branch = "wasmedge", features = [
+    "rt",
+    "time",
+    "macros",
+] }
+wasmedge-anna-driver = { path = "../../" }
```

Comment:
Can you put the driver into a crate instead of a relative path, so that this example can standalone? Thanks.